bionic/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:bionic
+LABEL maintainer="Shaun Jackman <sjackman@gmail.com>"
+LABEL name="Linuxbrew/bionic"
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends ca-certificates curl file g++ git locales make uuid-runtime \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && localedef -i en_US -f UTF-8 en_US.UTF-8 \
+    && useradd -m -s /bin/bash linuxbrew \
+    && echo 'linuxbrew ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
+
+USER linuxbrew
+WORKDIR /home/linuxbrew
+ENV PATH=/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH \
+    SHELL=/bin/bash
+
+RUN git clone https://github.com/Linuxbrew/brew.git /home/linuxbrew/.linuxbrew/Homebrew \
```

Comment:
Homebrew/brew

Reply:
Would it make sense to first merge this and then do a global replacement of those URLs, seeing that they're part of all Dockerfiles?

Reply:
You're a perfectionist but in a strange way :)
I prefer to merge things that work. If you want, you can do mass fix first and then we'll merge this one but with correct URL

Reply:
Heh, I'm just being economic.
But sure, I'll update it.